### ISSUE TITLE
Prepending .. in relative imports

### ISSUE THREAD

**Issue by ncordon** [bblfsh/python-driver#213]
Currently, for `from .some import relative`, we get a node with `..`:

<img width="1028" alt="relative" src="https://user-images.githubusercontent.com/5649971/64607757-75d08d80-d3c9-11e9-9e56-fde3adb54e0f.png">

And for `from ..some import relative`, we get two nodes with `..`:

![relativex2](https://user-images.githubusercontent.com/5649971/64607794-85e86d00-d3c9-11e9-9d44-be0317230b97.png)

It may be related to [this line](https://github.com/bblfsh/python-driver/blob/7f07c7625798c31965abced694d89160be2fd1af/driver/normalizer/util.go#L45=

**Comment by ncordon** [comment]
HT @smacker and @kuba--

**Comment by r0mainK** [comment]
@ncordon isn't that accurate though ?
```
from .some import relative
```
Means `relative`  is in the `some` file that is in the same directory as the current file. Whereas with:
```
from ..some import relative
```
Means `relative`  is in the `some` file that is in the parent directory of the directory containing the current file. So it makes sense that the first would have one `..` node, and the second two ?

**Comment by smacker** [comment]
All single dots are returned as double dots right now but splitting happens correctly. As a workaround, you can just post-process bblfsh results by replacing `..` with `.` to get the right relative path. (at least it worked for all cases I had):

<img width="1066" alt="Screenshot 2019-09-27 at 12 02 52" src="https://user-images.githubusercontent.com/406916/65761377-f1973d80-e11e-11e9-9a88-e4e63f5d5d75.png">

**Comment by r0mainK** [comment]
Yeah I did the same, and I'd say the double dot makes more sense to me then a single one, as if you were in the file and wanted to go to the import, you would move around doing `cd ../../some` etc.
